plugins/cilium-cni/cni-install.sh

```diff
@@ -174,33 +174,9 @@ EOF
 	;;
 
 "aws-cni")
-	cat > "${CNI_CONF_NAME}" <<EOF
```

Comment:
Some concerns here. If the file `10-aws.conflist.cilium_bak` does not exist on the node the Cilium CNI config will not be created on the node. Previously it was always created by Cilium. Is it possible to have a backup plan in case `10-aws.conflist.cilium_bak` does not exist and we create the file regardless?

Reply:
Good point. I'll check if I can get the old behavior back and still give people the opportunity to customize the necessary settings. Might also be interesting for people that want to disable the IPv4 NAT when running Cilium.

Reply:
Hey @aanm, sorry for taking so long to address your review comment. I got side-tracked by a few other things. I just submitted a change that restores the old behavior in case that the file does not exist. This seemed to be the most straightforward solution. It should work for both IPv4 and IPv6 clusters, on IPv6 clusters pods just won't have host-local IPv4 NAT available with the backup strategy.